BDArmory/FX/FuelLeakFX.cs

```diff
@@ -21,13 +22,37 @@ public static ObjectPool CreateLeakFXPool(string modelPath)
             return ObjectPool.CreateObjectPool(template, 10, true, true);
         }
 
+        public float drainRate = 1;
+        private int fuelLeft = 0;
         public float lifeTime = 20;
         private float startTime;
         private float disableTime = -1;
         private float _highestEnergy = 1;
+
+        PartResource fuel;
+        PartResource ox;
```

Comment:
Building this says that `FuelLeakFX.ox` and `FuelLeakFX.mp` are assigned but not used. You appear to be using local variables of the same name in `Update`.